### ISSUE TITLE
为什么我开始训练就把CPU跑到100%，显卡确一点占用都没有

### ISSUE THREAD

**Issue by lzg134679** [svc-develop-team/so-vits-svc#25]

**Comment by Miuzarte** [comment]
首先 请提供日志
我不知道你是不是一开始训练连步数都没报出来就说没吃GPU
如果跑了两百步还是完全没有一点GPU占用，就检查一下：linux上需要torch+cudnn两个依赖一起装，Windows则检查torch版本号带不带cu；然后不管什么平台都检查一下cuda device有没有设置成0
一般都是这两个问题

**Comment by lzg134679** [comment]
2023-03-14 16:53:10,725	44k	INFO	{'train': {'log_interval': 200, 'eval_interval': 800, 'seed': 1234, 'epochs': 10000, 'learning_rate': 0.0001, 'betas': [0.8, 0.99], 'eps': 1e-09, 'batch_size': 6, 'fp16_run': False, 'lr_decay': 0.999875, 'segment_size': 10240, 'init_lr_ratio': 1, 'warmup_epochs': 0, 'c_mel': 45, 'c_kl': 1.0, 'use_sr': True, 'max_speclen': 512, 'port': '8001', 'keep_ckpts': 3}, 'data': {'training_files': 'filelists/train.txt', 'validation_files': 'filelists/val.txt', 'max_wav_value': 32768.0, 'sampling_rate': 44100, 'filter_length': 2048, 'hop_length': 512, 'win_length': 2048, 'n_mel_channels': 80, 'mel_fmin': 0.0, 'mel_fmax': 22050}, 'model': {'inter_channels': 192, 'hidden_channels': 192, 'filter_channels': 768, 'n_heads': 2, 'n_layers': 6, 'kernel_size': 3, 'p_dropout': 0.1, 'resblock': '1', 'resblock_kernel_sizes': [3, 7, 11], 'resblock_dilation_sizes': [[1, 3, 5], [1, 3, 5], [1, 3, 5]], 'upsample_rates': [8, 8, 2, 2, 2], 'upsample_initial_channel': 512, 'upsample_kernel_sizes': [16, 16, 4, 4, 4], 'n_layers_q': 3, 'use_spectral_norm': False, 'gin_channels': 256, 'ssl_dim': 256, 'n_speakers': 200}, 'spk': {'nyaru': 0, 'huiyu': 1, 'nen': 2, 'paimon': 3, 'yunhao': 4}, 'model_dir': './logs\\44k'}
2023-03-14 16:53:10,725	44k	WARNING	E:\so-vits-svc-4.0 is not a git repository, therefore hash value comparison will be ignored.
2023-03-14 16:53:13,631	44k	INFO	Loaded checkpoint './logs\44k\G_0.pth' (iteration 1)
2023-03-14 16:53:14,100	44k	INFO	Loaded checkpoint './logs\44k\D_0.pth' (iteration 1)
2023-03-14 16:53:18,644	44k	INFO	====> Epoch: 1, cost 7.92 s
2023-03-14 16:53:23,179	44k	INFO	====> Epoch: 2, cost 4.53 s
2023-03-14 16:53:27,667	44k	INFO	====> Epoch: 3, cost 4.49 s
2023-03-14 16:53:32,230	44k	INFO	====> Epoch: 4, cost 4.56 s
2023-03-14 16:53:36,823	44k	INFO	====> Epoch: 5, cost 4.59 s
2023-03-14 16:53:42,271	44k	INFO	====> Epoch: 6, cost 5.45 s
2023-03-14 16:53:47,327	44k	INFO	====> Epoch: 7, cost 5.06 s
2023-03-14 16:53:51,899	44k	INFO	====> Epoch: 8, cost 4.57 s
2023-03-14 16:53:56,367	44k	INFO	====> Epoch: 9, cost 4.47 s
2023-03-14 16:54:01,019	44k	INFO	====> Epoch: 10, cost 4.65 s
2023-03-14 16:54:05,615	44k	INFO	====> Epoch: 11, cost 4.60 s
2023-03-14 16:54:10,152	44k	INFO	====> Epoch: 12, cost 4.54 s
2023-03-14 16:54:14,695	44k	INFO	====> Epoch: 13, cost 4.54 s
2023-03-14 16:54:19,286	44k	INFO	====> Epoch: 14, cost 4.59 s
2023-03-14 16:54:23,782	44k	INFO	====> Epoch: 15, cost 4.50 s
2023-03-14 16:54:28,382	44k	INFO	====> Epoch: 16, cost 4.60 s
2023-03-14 16:54:32,911	44k	INFO	====> Epoch: 17, cost 4.53 s
2023-03-14 16:54:37,469	44k	INFO	====> Epoch: 18, cost 4.56 s
2023-03-14 16:54:41,952	44k	INFO	====> Epoch: 19, cost 4.48 s
2023-03-14 16:54:46,480	44k	INFO	====> Epoch: 20, cost 4.53 s

**Comment by lzg134679** [comment]
日志没有报错，只有一条警告，训练也能正常进行
![5eab3c970a9bfeb7de3a4eb0f1af29d](https://user-images.githubusercontent.com/82492310/224959811-9ca67d80-4437-4f4e-a3a3-25979bdef07a.png)

**Comment by Miuzarte** [comment]
显存没占用确实是在用CPU跑，去[https://pytorch.org/get-started/locally/](https://pytorch.org/get-started/locally/)安装cu11.7的torch
其次  CPU跑也能几秒钟一个epoch，你训练集就几条？sovits可不能给你变魔术

然后查看显卡占用应该看cuda的数据
![image](https://user-images.githubusercontent.com/66856838/224960360-d9c45393-b2be-4df8-819f-9ec88e797663.png)

**Comment by Miuzarte** [comment]
然后如果在用一键包的话，请找这个一键包的作者询问问题

**Comment by Miuzarte** [comment]
更正：不是显存没占用，是数据集实在是太少了，占用上不去

**Comment by seq2193** [comment]
大佬你qq mail 洩漏了

**Comment by lzg134679** [comment]
![image](https://user-images.githubusercontent.com/82492310/224998698-9d1037e1-bc83-45f3-8454-297467eb9a04.png)
cuda也没有占用是不是能说明在用cpu跑
